lib/modules/datasource/terraform-module/index.ts

```diff
@@ -43,13 +49,39 @@ export class TerraformModuleDatasource extends TerraformDatasource {
       'terraform-module.getReleases()'
     );
 
+    const serviceDiscovery = await this.getTerraformServiceDiscoveryResult(
+      registryUrl
+    );
+    const registryHost = parseUrl(registryUrl)?.host;
+    if (registryHost === 'registry.terraform.io') {
```

Comment:
```suggestion
    if (registryUrl === defaultRegistryUrls[0]) {
```

Datasource layer will trim any trailing slashes